kajas/src/app/login/login.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="container">
+  <h1>Welcome to</h1>
+  <div class="logo">
+    <img src="../../assets/kajas_black.png" alt="Kajas Logo" />
+  </div>
+  <h3>"Where Art Finds its Home"</h3>
+
+  <form [formGroup]="loginForm" (ngSubmit)="onSubmit()">
+    <div class="input-group">
+      <input
+        type="email"
+        id="user_email"
+        formControlName="email"
+        placeholder="Email Address"
+      />
```

Comment:
### Remove unnecessary breaking of lines.
- Place line breaks in proper places.
[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27](url)